app/controllers/greetings_controller.rb

```diff
@@ -0,0 +1,51 @@
+class GreetingsController < ApplicationController
+  before_action :set_greeting, only: %i[show update destroy]
+
+  # GET /greetings
+  def index
+    greeting = Greeting.order('RANDOM()').first
+    render json: { message: greeting.message }
+  end
+
+  # GET /greetings/1
+  def show
```

Comment:
- [ ] As requested by previous reviewer, kindly fix all compilation problems so that your app can work correctly.
![image](https://user-images.githubusercontent.com/94551484/220836701-0667d631-8e1f-43ab-864d-ba0a3d1aca89.png)

- [ ] Your app still does not display greetings, kindly fix this please since this is the main requirement of theproject. 
![image](https://user-images.githubusercontent.com/94551484/220837487-c262fff4-bb5f-40a6-aa2f-80f2e233e165.png)